### PR TITLE
Fix property usage example issue in docs

### PR DESCRIPTION
**Pull request by chenjian2664**

### CODE REVIEW

**Comment by wgzhao** [comment]
Thanks.
Here it should use arrays, in the process of writing code, in the use of strings or array definitions, changed a few times, resulting in the documentation is not updated

**Comment by chenjian2664** [comment]
cc @findepi @mosabua

File: docs/src/main/sphinx/connector/clickhouse.rst [hunk 1]
```diff
@@ -79,7 +79,7 @@ Table property usage example::
     )
     WITH (
       engine = 'MergeTree',
-      order_by = 'id, birthday',
+      order_by = ARRAY['id, birthday'],
```
Comment (hashhar) [inline docs/src/main/sphinx/connector/clickhouse.rst:82]:
```suggestion
      order_by = ARRAY['id', 'birthday'],
```

Comment (chenjian2664) [reply docs/src/main/sphinx/connector/clickhouse.rst]: @hashhar  Thanks!

**Review by hashhar — APPROVED** [review]

**Pull request merged**